RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -109,7 +109,16 @@ extension View {
     @ViewBuilder
     func scrollableIfNecessaryWhenAvailable(_ axis: Axis = .vertical, enabled: Bool = true) -> some View {
         if enabled {
-            if #available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
+            // There is a bug in iOS 18 beta 5 (as of writing, future versions uncofirmed)
+            // where ViewThatFit crashes when used.
+            // This has been filed as FB14699941.
+            // Until this is fixed, we're rolling back to pre-iOS 16 behavior for this view.
+            // More information and discussion here: https://github.com/RevenueCat/purchases-ios/issues/4150
+            let iOSVersionWithCrash = OperatingSystemVersion(majorVersion: 18, minorVersion: 0, patchVersion: 0)
+            let isIOSVersionWithCrash = ProcessInfo.processInfo.isOperatingSystemAtLeast(iOSVersionWithCrash)
```

Comment:
note: I'm kinda going out of my way to not name this `iOS18` so that people don't get knocked for having references to unreleased software, just in case. I've seen Apple's code checks get really dumb.

Reply:
might even be worth extracting the `18` to a different variable just in case too

Reply:
`majorVersion: 6*3`

Reply:
`majorVersion: 1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1`

Reply:
😆